src/components/biggive-hero-image/biggive-hero-image.tsx

```diff
@@ -25,13 +28,18 @@ export class BiggiveHeroImage {
   /**
    * Header slug colour
    */
-   @Prop() slugColour: string;
+   @Prop() slugColour: brandColour;
 
   /**
    * Full URL of a logo image.
    */
    @Prop() logo: string = '';
 
+  /**
+   * Logo container height.
+   */
+    @Prop() logoHeight: number = 3;
```

Comment:
Not a blocker but we could make this more self documenting - I think this is right, let me know if I've misunderstood:
```suggestion
  /**
   * Logo container height selection. Numbers are not measurements.
   */
    @Prop() logoHeight: 1|2|3|4|5|6|7|8|9|10 = 3;
```

Reply:
Updated as per your suggestion